src/gsp.py

```diff
@@ -286,6 +313,10 @@ def get_truths_for_a_specific_gsp_old_route(
     user: Auth0User = Security(get_user()),
 ) -> List[GSPYield]:
     """Redirects old API route to new route /v0/solar/GB/gsp/{gsp_id}/pvlive"""
+
+    if is_fake:
+        make_fake_gsp_yields(gsp_ids=List[gsp_id], session=session)
```

Comment:
should this just be `[gsp_id]` not `List[gsp_ids]`

Reply:
Awesome! Thanks for this. Looks like it's worked!